db/config.cc

```diff
@@ -1121,6 +1130,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "Ignored if authentication tables already contain a super user password.")
     , auth_certificate_role_queries(this, "auth_certificate_role_queries", value_status::Used, { { { "source", "SUBJECT" }, {"query", "CN=([^,]+)" } } },
         "Regular expression used by CertificateAuthenticator to extract role name from an accepted transport authentication certificate subject info.")
+    , max_tablet_size(this, "max_tablet_size", liveness::LiveUpdate, value_status::Used, service::default_max_tablet_size, max_tablet_size_help_string())
```

Comment:
I guess _mb for ease of use and consistency with other options (though I prefer bytes myself).

Reply:
Oh and the help string needs to be a literal since we generate documentation by parsing this file.

Reply:
Maybe I can extend db config to have an option to dump the default value (if desired).

Reply:
Let's just skip it. Eventually tablet size will be determined dynamically by available disk.

Reply:
Ok.

Reply:
It needs to be an inline string, or the parser will be confused.

Reply:
And let's formulate it as average.